src/main/java/com/mongodb/jdbc/MongoJsonSchema.java

```diff
@@ -42,7 +50,7 @@ public ScalarProperties(String name, BsonTypeInfo type) {
     public Map<String, MongoJsonSchema> properties;
     public Set<MongoJsonSchema> anyOf;
     public Set<String> required;
-    public MongoJsonSchema items;
+    public Set<MongoJsonSchema> items;
```

Comment:
I don't think this type should be changing -- conceptually, we only maintain a single schema that describes all elements of an array, instead of allowing separate validators for values at different indexes.

When we see an array-style items schema, I think we want to merge the index schemas into a single `MongoJsonSchema` that is an `anyOf` of each of the item schemas